config/initializers/productization.rb

```diff
@@ -1 +1,3 @@
-Vmdb::Productization.new.prepare
+Rails.application.config.to_prepare do
```

Comment:
@simaishi I would think that if you were able to swap back and forth with productization symlinked/unlinked and verified that it's called only once in dev/prod mode and serves the right images, it would be a good solution.